catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -89,115 +86,35 @@ function RevisionInfo({ revision, bucket, name, path }) {
-                                      borderRadius="borderRadius"
-                                      height={16}
-                                      width="90%"
-                                    />
-                                  ),
-                                },
-                                msg,
-                              )}
-                              <br />
-                              {AsyncResult.case(
-                                {
-                                  Ok: (v) => (
-                                    <span className={cx(classes.line, classes.mono)}>
-                                      {v}
-                                    </span>
-                                  ),
-                                  _: () => (
-                                    <Skeleton
-                                      component="span"
-                                      display="inline-block"
-                                      borderRadius="borderRadius"
-                                      height={16}
-                                      width="95%"
-                                    />
-                                  ),
-                                },
-                                hash,
-                              )}
-                            </span>
-                          }
-                        />
-                      </M.ListItem>
-                    )
-                  }}
-                </Data>
-              ))
-              if (isTruncated) {
-                revList.unshift(
-                  <M.ListItem key="__truncated">
-                    <M.ListItemText
-                      primary="Revision list is truncated"
-                      secondary="Latest revisions are not shown"
-                    />
-                    <M.ListItemSecondaryAction>
-                      <M.Icon>warning</M.Icon>
-                    </M.ListItemSecondaryAction>
-                  </M.ListItem>,
-                )
-              }
-              return revList
-            },
+            Ok: R.map((r) => (
+              <M.ListItem
+                key={r.hash}
+                button
+                onClick={close}
+                selected={r.id === revision}
+                component={RRLink}
+                to={urls.bucketPackageTree(bucket, name, r.id, path)}
+              >
+                <M.ListItemText
+                  primary={
+                    <>
+                      <span className={classes.mono}>{r.id}</span>
+                      {' | '}
+                      {dateFns.format(r.modified, 'MMMM do yyyy - h:mma')}
+                    </>
+                  }
+                  secondary={
+                    <span className={classes.secondaryText}>
+                      <span className={classes.line}>
+                        {r.message || <i>No message</i>}
+                      </span>
+                      <br />
```

Comment:
You already have `.line` class and it's `nowrap`ed, I think you can add `display: block` instead of `<br />`